vocode/streaming/telephony/conversation/twilio_call.py

```diff
@@ -156,5 +162,4 @@ async def handle_ws_message(self, message) -> Optional[PhoneCallWebsocketAction]
 
     def mark_terminated(self):
         super().mark_terminated()
-        asyncio.create_task(self.telephony_client.end_call(self.twilio_sid))
-
+        # asyncio.create_task(self.telephony_client.end_call(self.twilio_sid))
```

Comment:
this is commented out because it immediately disconnects the transferred call if its left in.

`response = self.twilio_client.calls(twilio_sid).update(status="completed")`

would love some insight here towards a solution here.  I thought about using redis to set some kind of conditional flag telling it not to disconnect the call if its being transferred.

any maintainers willing to point me in the right direction?

Reply:
yup agreed, i think there should be a boolean flag for whether or not the call should be completely disconnected (this is useful in cases where we want to end the ws but not the call eg transfer or dtmf)

Reply:
+1 we can maybe leverage the `active` prop on `StreamingConversation` here. if `active`, we don't end the call

Reply:
how do you want the `TransferCall` action to communicate with `StreamingConversation` to keep the conversation `active` when its being transferred?

```
elif data["event"] == "stop":
            self.logger.debug(f"Media WS: Received event 'stop': {message}")
            self.logger.debug("Stopping...")
            return PhoneCallWebsocketAction.CLOSE_WEBSOCKET
```